test/fixtures/README.md

```diff
@@ -0,0 +1,89 @@
+# Fixtures Directory
+
+This directory contains test fixtures for various certificate types and schema formats used across the Material Identity ecosystem. These fixtures serve as reference examples for implementation, testing, and validation purposes.
+
+## Directory Structure
+
+Fixtures are organized by certificate type and schema version:
+
+```
+fixtures/
+├── README.md                       # This file
+├── coa/                            # Certificate of Analysis fixtures
+│   ├── v0.x.y/                     # Version-specific fixtures
+│   │   ├── example_001.json        # Example certificate in JSON format
+│   │   ├── example_001.pdf         # Corresponding PDF rendering
+│   │   └── ...                     # More example certificates
+│   └── ...                         # Other versions
+│
+├── en10168/                        # EN10168 certificate fixtures
+│   ├── v0.4.1/                     # Version-specific fixtures
+│   │   ├── valid_certificate_1.json # Example certificate in JSON format
+│   │   ├── valid_certificate_1.pdf  # Corresponding PDF rendering
+│   │   └── ...                     # More example certificates
+│   └── ...                         # Other versions
+│
+├── forestry/                       # Forestry certificate fixtures
+│   └── ...                         # Structure similar to above
+│
+└── metals/                         # Metal industry certificate fixtures
+    └── ...                         # Structure similar to above
+```
+
+## Purpose
+
+The fixtures in this directory provide:
+
+1. **Reference Implementations**: Complete, valid examples of certificates conforming to each schema
+2. **Test Data**: Data for automated testing of parsers, validators, and renderers
+3. **Developer Guidance**: Practical examples of how to structure data for each schema type
+4. **Validation Examples**: Sample documents to validate schema changes against
+5. **Rendering Examples**: Visual reference for how each certificate should be presented
+
+## Usage Guidelines
+
+### Testing
+
+Use these fixtures for testing implementations:
+
+```javascript
+// Example test code
```

Comment:
Wouldn't it be better to show how to use them with the current test setup?

Reply:
Yes.